fix(digital): grammar and vocabulary (#399)
dergigi/dergigi.github.io: collections/bitcoin/_posts/2022-10-02-bitcoin-is-digital-scarcity.md

ever so slightly for several weeks and when I logged on that morning to
do business as usual the trade channel was in a state of chaos.
Stormshields were going for 20 SoJ, other higher value items were

> impossible to obtain. The heydays were over, and everyone knew it. The

Reviewer: ```suggestion
impossible to obtain. The heydays were over and everyone knew it. The
```

I wouldn't alter the original, it's a quote after all.

more valuable than its appearance would suggest. Men have given much to
possess it."*

> Once people home in on one form of money, they do indeed go through

Reviewer: [Today I learned!](https://www.merriam-webster.com/words-at-play/home-in-or-hone-in)

Author: I had to double-check that myself  ;)

their fellow residents: entertainment, education, consulting, business
management, news, and yes, even banking.

> Consequently, many people ditched their real-life job to focus fully

Reviewer: ```suggestion
As a consequence, many people ditched their real-life job to focus fully
```

the digital world of the bitcoin network. How is this any different from
the various in-game monies that emerged before it?

> It is different because both the Bitcoin environment as well as its monetary units

Reviewer: ```suggestion
It is different because both the environment as well as its monetary units
```

[^fn-diamonds]: Diamonds are one example of a real-world good that used to be scarce but can now be farmed. Thanks to human ingenuity, diamonds can be produced artificially. To quote the [U.S. National Minerals Information Center](https://web.archive.org/web/20221002111535/https://www.usgs.gov/centers/national-minerals-information-center/industrial-diamond-statistics-and-information): "Synthetic industrial is superior to its natural diamond counterpart because it can be produced in unlimited quantities, and, in many cases, its properties can be tailored for specific applications. Consequently, manufactured diamond accounts for more than 90% of the industrial diamond used in the United States."

When it comes to in-game currencies, the company that is running the game

> servers might go bankrupt or decide to shut down operations for other reasons.

Reviewer: ```suggestion
servers might go bankrupt or decides to shut down operations for other reasons.
```

The company [...] decides to shut down ...

Author: I respectfully disagree, it's the "might" in the sentence that changes it: The company might [go bankrupt or] decide to shut down operations

nodes will reject them. Miners extend the arena, and in doing so, are
rewarded with sats---the most precious objects in the world of Bitcoin.
Anyone can participate in block production. All that is required is an

> energy source, a computer, and a communications channel.

Reviewer: ```suggestion
energy source and a communications channel.
```

No. https://www.youtube.com/watch?v=y3dqhixzGVo

these arenas were always *virtual* worlds. Designed, controlled, and
maintained by central authorities. Authorities with the power to change
the rules, something that they will always do, either out of

> self-interest or coercion. As we have seen, if a central authority can

Reviewer: ```suggestion
self-interest or emergency. As we have seen, if a central authority can
```

Jaron Lanier quipped that "information is not something that exists."
I disagree. Information that can only exist because of expensive

> physical processes has a certain reality to it. It is incontrovertible

Reviewer: 👍

3.  "the arena" ceases to exist, either because of bankruptcy,
    intervention, or collapse.

> The combination of non-simulative proof-of-work with cheap, independent

Reviewer: ```suggestion
The combination of non-simulatable proof-of-work with cheap, independent
```

https://www.wordnik.com/words/simulatable

mechanism to ensure that the past can't be altered cheaply and to ensure that
future issuance can't be farmed efficiently.

> Consequently, Bitcoin's energy consumption is not a by-product of the

Reviewer: ```suggestion
Consequently, Bitcoin's energy consumption is not a byproduct of the
```

https://www.merriam-webster.com/dictionary/by-product

> variants: byproduct

In short: bitcoin can't be printed because it is automatically issued
over time. The system can't be cheated because energy can't be copied.
Proof-of-work can't be simulated, and the difficulty adjustment forbids

> the "farming" of blocks. Anyone can participate, instantiate, and

Reviewer: ```suggestion
the "farming" of blocks. Anyone can participate in, instantiate, and
```

> validate.

Reviewer: I don't really like this change, because I the "everything" is kind of important. 

You can participate in everything. You can instantiate everything. And you can validate everything.

Author: Agree on instantiate and validate, but you can't really participate in "everything"? You can participate in the Bitcoin network, but you can't practically participate in every aspect of it?

Reviewer: You can, and that's the whole point. You can mine, you can generate keys, you can run a node.

Author: Ok, I was coming at it from the angle of "participate in every single transaction, mine every block" etc

> validate.

Reviewer: ```suggestion
validate everything.
```
